lib/fetch/formdata.js

```diff
@@ -158,29 +159,32 @@ class FormData {
     webidl.brandCheck(this, FormData)
 
     return makeIterator(
-      () => this[kState].map(pair => [pair.name, pair.value]),
+      () => this[kState],
       'FormData',
-      'key+value'
+      'key+value',
+      'name', 'value'
     )
   }
 
   keys () {
     webidl.brandCheck(this, FormData)
 
     return makeIterator(
-      () => this[kState].map(pair => [pair.name, pair.value]),
+      () => this[kState],
       'FormData',
-      'key'
+      'key',
+      'name', 'value'
     )
   }
 
   values () {
     webidl.brandCheck(this, FormData)
 
     return makeIterator(
-      () => this[kState].map(pair => [pair.name, pair.value]),
+      () => this[kState],
```

Comment:
This is the reason why the performance of the loop is poor.

Reply:
I actually think we need to create a copy...